client/my-sites/customer-home/main.jsx

```diff
@@ -153,18 +139,12 @@ const Home = ( {
 		);
 	}
 
-	// While fetching all the data, show placeholder.
-	// After fetching all the data:
-	//  - For eCommerce & eCommerce trial, don't show placeholder.
-	//  - For Woo Express Trial, Essential & Performance, show placeholder while redirecting to WooCommerce Home (search for `wc-admin` under customer-home/controller.jsx).
+	// Ecommerce Plan's Home redirects to WooCommerce Home, so we show a placeholder
+	// while doing the redirection.
```

Comment:
Just checking whether this comment is spread into two lines on purpose.

Reply:
@andregardi is following the max 80 chars per line rule, like the punch cards in the hey days.
<img width="681" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1287077/1e0052fa-5d8b-484a-aa50-ce8a63b505cc">

Reply:
Thanks, Yan! Makes sense. 🙂